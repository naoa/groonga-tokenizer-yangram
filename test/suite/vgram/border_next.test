register tokenizers/yangram

table_create #vgram_words TABLE_HASH_KEY ShortText
load --table #vgram_words
[
{"_key": "情報"}
]
tokenize TokenYaVgram "の情報A処理" NormalizerAuto --mode ADD
tokenize TokenYaVgram "情報" NormalizerAuto --mode GET
