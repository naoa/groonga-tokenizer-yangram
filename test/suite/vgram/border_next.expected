register tokenizers/yangram
[[0,0.0,0.0],true]
table_create #vgram_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table #vgram_words
[
{"_key": "情報"}
]
[[0,0.0,0.0],1]
tokenize TokenYaVgram "の情報A処理" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "の情報",
      "position": 0
    },
    {
      "value": "情報",
      "position": 1
    },
    {
      "value": "報",
      "position": 2
    },
    {
      "value": "a",
      "position": 3
    },
    {
      "value": "処理",
      "position": 4
    },
    {
      "value": "理",
      "position": 5
    }
  ]
]
tokenize TokenYaVgram "情報" NormalizerAuto --mode GET
[[0,0.0,0.0],[{"value":"情報","position":0}]]
